.github/workflows/ci.yaml

```diff
@@ -74,18 +74,23 @@ jobs:
     - uses: jpkrohling/setup-operator-sdk@v1.1.0
       with:
         operator-sdk-version: v1.28.0
-    - name: Install qemu
-      continue-on-error: false
+    - name: Install podman v4
       run: |
-        sudo apt-get update
-        sudo apt-get install -y qemu-user-static
+        ubuntu_version='22.04'
```

Comment:
@ebaron @andrewazores Any other clean ways to install podman v4 on 22.04 ubuntu?

Not very familar with it but seems like its stuck with podman 3.4 and I have to use a suggested workaround.

Reply:
https://github.com/containers/podman/issues/14302

Reply:
https://devicetests.com/install-podman-4-ubuntu-22-04

Reply:
I'm really not familiar with Ubuntu's packaging and repos, but using an `opensuse.org` repository looks reasonable/sane/safe. If podman v4 is really needed and it isn't normally shipped, this seems like a fine workaround.

Reply:
okayy sounds good! Just testing on my fork now..

Reply:
https://github.com/containers/podman/issues/14302#issuecomment-1310886397

There is also that option, but I don't have much of an opinion either way. I think using the prebuilt unofficial "PPA" hosted on opensuse infra is fine - I don't know if Homebrew will actually try to build things on the CI runner, which would be far less ideal.

Reply:
Seems like I was able to get `podman 4.4.6` using opensuse repo.

https://github.com/tthvo/cryostat-operator/actions/runs/5969221583/job/16194662780?pr=6